app/forms.py

```diff
@@ -16,17 +17,22 @@ class RegistrationForm(FlaskForm):
     submit = SubmitField('Sign up')
 
     @staticmethod
-    def validate_army_name(army_name):
+    def validate_army_name(self, army_name):

```

Comment:
Why do you use `self` if it's a static method?

Reply:
when i tried without self i got an error